### ISSUE TITLE
Image fields don't work in velocity with dotAssets

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#22704]
1. Create a content type `testing`
    - Title
    - Image (Image Field)
2. Create a content object of `testing` - use DnD to upload a dotAsset to the field
3. run this against your dotCMS instance and you can see that the velocity content object is unable to use/retrieve the image field.

```
curl -u admin@dotcms.com:admin -XGET https://demo.dotcms.com/api/vtl/dynamic/  \
-H "Content-Type:text/plain" \
-d '
    #set($test = $dotcontent.pull("+contentType:testing",  100,"").get(0))
    fails: $test.image
    fails: $test.getRaw("image")
    works: $test.contentObject.image
'
```

Output:
```
    fails: $test.image
    fails: $test.getRaw("image")
    works: fbdf777876229e952939a9dff93c3932
```
Bonus crappiness - the info is messed up for the image field as well:

![Screen Shot 2022-08-09 at 10 22 46 AM](https://user-images.githubusercontent.com/934364/183676367-f0fe84d1-e8e8-419d-a9a0-387e1e18d2d8.png)

**Comment by wezell** [comment]
dupe: #22704

**Comment by nollymar** [comment]
PR: #22710

**Comment by alfredo-dotcms** [comment]
console output
![image](https://user-images.githubusercontent.com/37185433/185238305-a5216ca4-b570-4d40-b5b5-ff572ccc6e9f.png)

![image](https://user-images.githubusercontent.com/37185433/185238369-d631358a-65f2-4869-b539-20ca4fec923e.png)
broken img url

**Comment by alfredo-dotcms** [comment]
the img url in the UI is broken, but in the console is good

**Comment by fabrizzio-dotCMS** [comment]
here is the fix for the displayed title  https://github.com/dotCMS/core/pull/22831

**Comment by fabrizzio-dotCMS** [comment]
I also verified that the VTL renders the expected values that were not present before

**Comment by wezell** [comment]
Even though Alfredo's question above is valid and probably an issue, that was not the point of the original card.  Passing for now and we can revisit in the future.

**Comment by nollymar** [comment]
**Internal QA:** The problem with the title has been fixed: 

![Screen Shot 2022-08-24 at 6 29 32 PM](https://user-images.githubusercontent.com/8741395/186541519-aac2a0cc-caf0-46fb-8316-ed1e9c4a65e7.png)


and after running the curl test I get this output that is expected: 

```
curl -u admin@dotcms.com:2#GemAb?DsHzp8my -XGET http://localhost:8080/api/vtl/dynamic/  \
> -H "Content-Type:text/plain" \
> -d '
>     #set($test = $dotcontent.pull("+contentType:testing",  100,"").get(0))
>     fails: $test.image
>     fails: $test.getRaw("image")
>     works: $test.contentObject.image
> '

    fails: com.dotcms.rendering.velocity.viewtools.content.BinaryMap@7a64a7e5[name=isalnds-french-polynesia.jpeg,size=100.4 K,rawUri=/contentAsset/raw-data/2a4fb69a-1f79-4bef-be47-20dec8669c78/asset,resizeUri=/contentAsset/image/2a4fb69a-1f79-4bef-be47-20dec8669c78/asset/filter/Resize,thumbnailUri=/contentAsset/image/2a4fb69a-1f79-4bef-be47-20dec8669c78/asset/filter/Thumbnail,file=/Users/nollymarlonga/workspace-dotcms/tomcat9/webapps/ROOT/assets/8/d/8d429bc7-8b44-49fb-9e3b-8f1689a64480/asset/isalnds-french-polynesia.jpeg]
    fails: com.dotcms.rendering.velocity.viewtools.content.BinaryMap@7a64a7e5[name=isalnds-french-polynesia.jpeg,size=100.4 K,rawUri=/contentAsset/raw-data/2a4fb69a-1f79-4bef-be47-20dec8669c78/asset,resizeUri=/contentAsset/image/2a4fb69a-1f79-4bef-be47-20dec8669c78/asset/filter/Resize,thumbnailUri=/contentAsset/image/2a4fb69a-1f79-4bef-be47-20dec8669c78/asset/filter/Thumbnail,file=/Users/nollymarlonga/workspace-dotcms/tomcat9/webapps/ROOT/assets/8/d/8d429bc7-8b44-49fb-9e3b-8f1689a64480/asset/isalnds-french-polynesia.jpeg]
    works: 2a4fb69a-1f79-4bef-be47-20dec8669c78
```

**Comment by bryanboza** [comment]
Fixed, tested on release-22.09 // Docker // FF 

Following the steps I'm getting the same results that the @nollymar's comment
